src/main/java/net/tropicraft/core/common/entity/underdasea/atlantoku/EntityTropicraftWaterBase.java

```diff
@@ -99,6 +103,20 @@ public void entityInit() {
 	@Override
 	public void onLivingUpdate() {
 		super.onLivingUpdate();
+		//setDead();
+		if(this instanceof IPredatorDiet && world.getDifficulty().equals(EnumDifficulty.PEACEFUL)) {
```

Comment:
Should this be && !world.getDifficulty().equals(EnumDifficulty.PEACEFUL) ?

Reply:
no